src/redux/services/terminalShells.ts

```diff
@@ -28,6 +28,7 @@ const detectAvailableUnixShells = (osPlatform: NodeJS.Platform): ShellsMapType =
     });
   } else {
```

Comment:
Here we should check the same as for the `darwin` platform and only add the `zsh` and `bash` if they are found in `/etc/shells` folder

Reply:
Got it going 
![image](https://user-images.githubusercontent.com/65275144/187885402-c3dcea98-dd6f-452c-aa94-c3bc809455ba.png)